src/QualificationRunner.Core/Services/QualificationRunner.cs

```diff
@@ -172,6 +175,11 @@ private Section[] copySectionContents(Section[] sections)
          return sections.Select(copySectionContent).ToArray();
       }
 
+      private void copyContentSubFolders()
+      {
+         copySubDirectory(absolutePathFrom(_runOptions.ConfigurationFolder, CONTENT_FOLDER), _runOptions.ContentFolder);

```

Comment:
Also copying all sub folders from the content folder as input. Not that this folder does not necessarily exist (not enforced in anyways). But if we follow the conventions that we have, it will be copied automatically. That's why we had said we would define a prop in the qualification plan...but this seems overkill